MobiFlight/CustomDevices/CustomDevice.cs

```diff
@@ -131,6 +127,11 @@ public class CustomDevice
         /// </summary>
         public List<MessageType> MessageTypes = new List<MessageType>();
 
+        /// <summary>
+        /// Base path for custom firmware
+        /// </summary>
+        public String BasePath;
```

Comment:
Same comment as above. It'd be nice to calculate this instead of have it come from the JSON, unless I'm misunderstanding?

Reply:
it's not coming from the JSON - it can be a readonly property